examples/01_template/plot_register.py

```diff
@@ -33,18 +33,23 @@
     os.makedirs(write_dir)
 
 ##############################################################################
-# Affine registration
+# Create the template
 # -------------------
 from sammba.registration import anats_to_common
 
 affine_register = anats_to_common(retest.anat, write_dir, 400, caching=True)
 
 ##############################################################################
-# We set caching to True, so that this step computations are not restarted.
+# We set caching to True, so that the computations are not restarted.
 # The paths to the registered images can be accessed easilly
 registered_anats = affine_register.registered
 print(registered_anats)
 
+##############################################################################
+#Assess the template
```

Comment:
Could you please add a space between # and Asses ? The formatting has an impact on the html built documentation

Reply:
You may also need to add # --- with as many hyphens as necessary to underline the whole sentence. This will make *Asses the template* appears as a subsection